proofs/libsectorbuilder/cgo_bindings.go

```diff
@@ -403,3 +404,52 @@ func GeneratePoSt(
 
 	return proofs, goUint64s(resPtr.faults_ptr, resPtr.faults_len), nil
 }
+
+// VerifyPieceInclusionProof returns true if the piece inclusion proof is valid
+// with the given arguments.
+func VerifyPieceInclusionProof(sectorSize uint64, pieceSize uint64, commP [32]byte, commD [32]byte, proof []byte) (bool, error) {
+	commDCBytes := C.CBytes(commD[:])
+	defer C.free(commDCBytes)
+
+	commPCBytes := C.CBytes(commP[:])
+	defer C.free(commPCBytes)
+
+	pieceInclusionProofCBytes := C.CBytes(proof)
+	defer C.free(pieceInclusionProofCBytes)
+
+	resPtr := (*C.sector_builder_ffi_VerifyPieceInclusionProofResponse)(unsafe.Pointer(C.sector_builder_ffi_verify_piece_inclusion_proof(
```

Comment:
@ZenGround0 - All of these C symbols are defined in a single C archive file, which is called `libsector_builder_ffi.a`, and are automatically generated as part of the `sector-builder-ffi` Rust crate, which re-exports stuff from the `rust-fil-proofs-ffi` crate. 

TLDR: The prefix is not a thing that we have the ability to change. cc @sidke

Reply:
Is renaming the sector-builder-ffi rust crate to something more reflective of its functionality something you could do?  I'm not saying it is important at all right now but want to make sure we're on the same page that it is tech debt that we'd prefer to correct.

Reply:
I think we may be speaking past each other here. I'll unpack things further...

`go-filecoin` links a single C archive file called `libsector_builder_ffi.a` with a header file `libsector_builder_ffi.h`. This archive/headerfile pair contains all of the sector builder and sector builder-nonspecific functions. The `sector_builder_ffi` prefix is a C convention; symbols are global in C, so libraries prefix their function names with the library name in order to avoid collision with other libraries. The source library in this case is `sector-builder-ffi`.

If we split the sector builder-nonspecific stuff into a separate archive file, go-filecoin (and any other non-Rust node implementation) would need to link in two libraries (each containing the Rust stdlib and other dependencies), each with their own headerfile. We'd need to modify go-filecoin to include a new submodule dependency on `rust-fil-proofs-ffi` (for the sector builder-nonspecific stuff), and we'd need to sure that it never used a version of that submodule which was incompatible with the `rust-fil-proofs-sector-builder` submodule it was pointed at. 

In all, combining the two sets of functions into a single C library keeps that archive file as small as possible. It reduces the complexity of linking the sector builder in to a node (there are no sector builder consumers who don't also want to verify PoSt and seal proofs) to a minimum.

Does that make sense?

Reply:
I think understand what you are saying.  Let me summarize to make sure I understand.  What I want is for all of the places that say `sector_builder_ffi_(PoSt|PIP|verifySeal)` to instead say `proofs_ffi_(PoSt|PIP|verifySeal)`.  We can't do that without (1) linking two libraries and duplicating standard libs, or (2) renaming the rust sector builder source library `proofs-ffi`.  Neither of these options is better than just keeping the imperfectly scoped but functional `sector_builder_ffi` prefix, so there is no plan to change the prefix.

Reply:
Yessiree! You got it.